pkg/custom_detectors/custom_detectors.go

```diff
@@ -101,10 +102,6 @@ func (c *CustomRegexWebhook) FromData(ctx context.Context, verify bool, data []b
 	close(resultsCh)
 
 	for result := range resultsCh {
-		// NOTE: I don't believe this is being set anywhere else, hence the map assignment.
-		result.ExtraData = map[string]string{
-			"name": c.GetName(),
-		}
```

Comment:
This was the reason the response result wasn't being printed! 😛 @zricethezav

Reply:
Let me know if the name is something we want to print. I can make the change to append it.

Reply:
@kashifkhan0771 not sure if you saw the issue that came in which @rgmz mentioned, but do you think you could include the name back. Thanks.
https://github.com/trufflesecurity/trufflehog/issues/3567

Reply:
Yeah sure

Reply:
Added back: https://github.com/trufflesecurity/trufflehog/pull/3569